src/puppet-wechat.ts

```diff
@@ -82,6 +80,7 @@ type PuppetWeChatOptions = PUPPET.PuppetOptions & {
   launchOptions? : LaunchOptions
   stealthless?   : boolean
   uos?           : boolean
+  token?           : string
```

Comment:
Could you please explain what's this `token` for? It seems that it does not been referenced in the new code.